### PR TITLE
Add Postgres 15 testing for migrations

### PR DESCRIPTION
**Pull request by nschonni**
Not sure if the `14.5` should be changed to the floating `14` to use the latest, but I used that during the conversion because that is what CircleCI is still using

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

**Comment by shleeable** [comment]
Floating `14`  sounds good.

File: .github/workflows/test-migrations-one-step.yml [hunk 1]
```diff
@@ -23,9 +23,17 @@ jobs:
     needs: pre_job
     if: needs.pre_job.outputs.should_skip != 'true'
 
+    strategy:
+      fail-fast: false
+
+      matrix:
+        postgres:
+          - 14.5
```
Comment (nschonni) [inline .github/workflows/test-migrations-one-step.yml:31]:
```suggestion
          - 14
```

File: .github/workflows/test-migrations-two-step.yml [hunk 2]
```diff
@@ -23,9 +23,17 @@ jobs:
     needs: pre_job
     if: needs.pre_job.outputs.should_skip != 'true'
 
+    strategy:
+      fail-fast: false
+
+      matrix:
+        postgres:
+          - 14.5
```
Comment (nschonni) [inline .github/workflows/test-migrations-two-step.yml:31]:
```suggestion
          - 14
```

**Review by nschonni — COMMENTED** [review]
These suggestions can be added if the latest version of 14 should be tested against

**Comment by shleeable** [comment]
Let's 14/15.

**Comment by nschonni** [comment]
Realizing that the CircleCI images [don't have floating major versions](https://github.com/CircleCI-Public/cimg-postgres/pull/33), just Major.Minor https://circleci.com/developer/images/image/cimg/postgres
I guess 14.5 could at least go to 14.6, but the 14.7 release from last month isn't out for their image yet https://github.com/CircleCI-Public/cimg-postgres/pull/73

**Comment by nschonni** [comment]
I rebased now that the other PR updated to the floating 14 version. Could tweak this a bit to give it a better name like ` Test one step migrations / test (Postgres 15) (pull_request)` instead of `15-apline` in the job name

**Review by renchap — APPROVED** [review]
Good for me, floating major versions should be preferred here.

But should we test with more major versions? For example we are still running PG 12 on mastodon.social.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**